hub/api/dataset.py

```diff
@@ -101,6 +102,9 @@ def __iter__(self):
         for i in range(len(self)):
             yield self[i]
 
+    def to_pytorch(self, transform=None, workers=1):
+        return _to_pytorch(self, transform, workers=workers)
```

Comment:
it would be nice if this was the only thing in the API, and the rest of it is in `hub.core`

Reply:
At this point we might as well have `to_pytorch = _to_pytorch`.

I prefer that we extract the relevant parameters from `self` and hand them off to helpers. If we pass around `self` we might as well just have it on the Dataset.

Reply:
true yeah `_to_pytorch` ideally shouldn't have access to Dataset, this would be a circular dependency

Reply:
scratch that, i disagree, this is fine